proto/cluster.proto

```diff
@@ -197,6 +221,11 @@ message ClusterSpec {
   HeadGroupSpec head_group_spec = 1 [(google.api.field_behavior) = REQUIRED];
   // Optional. The worker group configurations
   repeated WorkerGroupSpec worker_group_spec = 2;
+  // EnableInTreeAutoscaling indicates whether operator should create in tree autoscaling configs
+	bool enableInTreeAutoscaling = 3;
```

Comment:
Is the field required?

Reply:
No, the cluster have to have head nodes, workers are optional